lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb

```diff
@@ -58,6 +58,12 @@
                             "to": ["approving"]
                         }
                     ]
+                }, {
+                    "name": "leave_a_comment",
```

Comment:
This is a new action, right, not a state? The way I read this change is that `leave_a_comment` is a new action that can be triggered from three current states, and the new action doesn't trigger a state change. Have you tested this in the UI to ensure it does what you expect, @jeremyf?